core/src/main/java/io/grpc/internal/AutoConfiguredLoadBalancerFactory.java

```diff
@@ -39,13 +42,12 @@
 import io.grpc.internal.ServiceConfigUtil.LbConfig;
 import java.util.ArrayList;
 import java.util.Collections;
-import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.logging.Logger;
 import javax.annotation.Nullable;
 
-@SuppressWarnings("deprecation") // migrate to AutoConfiguredLoadBalancerFactory2 is required
+@SuppressWarnings("deprecation") // after migrated to 2, we can deprecate it
```

Comment:
What deprecated usages are suppressed?

Reply:
`LoadBalancer.ATTR_LOAD_BALANCING_CONFIG`. those files look like changed in this PR, but those files are just swapped from 2 to 1. i'll update the comment since the 2 doesn't make sense anymore